source/jobs.c

```diff
@@ -664,6 +738,22 @@ JobsStatus_t Jobs_StartNext( char * buffer,
     return ret;
 }
 
+size_t Jobs_StartNextMsg( const char * clientToken,
+                          size_t clientTokenLength,
+                          char * buffer,
+                          size_t bufferSize )
+{
+    size_t start = 0U;
+
+    if( ( clientToken != NULL ) && ( clientTokenLength > 0U ) && ( bufferSize >= 18U + clientTokenLength ) )
+    {
+        strnAppend( buffer, &start, bufferSize, "{\"clientToken\":\"", sizeof( "{\"clientToken\":\"" ) - 1 );
```

Comment:
Part of me thinks that these `clientToken` strings should be `#define` values?

Reply:
Will update in next commit